pages/git-protocol.md

```diff
@@ -21,7 +23,9 @@ A guide for programming within version control.
 * Perform work in a feature branch.
 * Rebase frequently to incorporate upstream changes.
 * Use a [pull request] for code reviews.
+* [Sign your commits with a GPG key](https://help.github.com/categories/gpg/).
```

Comment:
I bet @jacobian would have some thoughts on this. For now, how about we say that it's recommended?